recipes/fwdpp/meta.yaml

```diff
@@ -0,0 +1,35 @@
+about:
+    home: 'https://www.github.com/molpopgen/fwdpp'
+    license: "https://raw.githubusercontent.com/molpopgen/fwdpp/master/LICENSE"
+    summary: "A C++ template library for forward-time population genetic simulation. (doi:  10.1534/genetics.114.165019)"
```

Comment:
Yes.  I'm going to leave the LD_LIBRARY_PATH, though, for reasons mentioned above.  OS X/conda in particular doesn't "just find" the runtime libs, IME.